src/main/scala/is/hail/methods/LDMatrix.scala

```diff
@@ -58,14 +59,19 @@ object LDMatrix {
           normalizedBlockMatrix.rowsPerBlock, normalizedBlockMatrix.colsPerBlock).toIndexedRowMatrix()
     }
     else {
-      indexedRowMatrix = (normalizedBlockMatrix multiply normalizedBlockMatrix.transpose)
+      import is.hail.distributedmatrix.DistributedMatrix.implicits._
+      val dm = DistributedMatrix[BlockMatrix]
+      import dm.ops._
+      indexedRowMatrix = (normalizedBlockMatrix * normalizedBlockMatrix.transpose)
         .toIndexedRowMatrix()
```

Comment:
This is a good point. I think that early on I decided this made sense since it would be a natural format for people to look at some rows in Spark or something, but it's not clear this should actually be this way. Probably should redo as our Breeze Block matrix in 0.1